python/src/ApproximationAlgorithmImplementationFactory_doc.i.in

```diff
@@ -1,4 +1,4 @@
-%feature("docstring") OT::ApproximationAlgorithmImplementation
+%feature("docstring") OT::ApproximationAlgorithmImplementationFactory
```

Comment:
Isn't it strange to mention "Implementation" in the Python doc?

Reply:
Why not?

Reply:
OK, never mind.